.github/sync/RELEASE.md

```diff
@@ -311,7 +321,7 @@ Currently, there are only 2 activities we use rever for, (1) aggregating the aut
 
 10. Update Release Issue to include a link to the Release PR.
```

Comment:
I'm seeing some capitalization inconsistencies with Release Issue and Release PR. Not a critical thing, but thought I'd point it out.

Reply:
Release branch appears to be all lowercase throughout. Any reason to your knowledge that these are capitalized?